cypress/e2e/eholdings/filter-by-tags-is-hidden.cy.js

```diff
@@ -0,0 +1,57 @@
+import { Permissions } from '../../support/dictionary';
+import { EHoldingsSearch } from '../../support/fragments/eholdings';
+import { AssignedUsers } from '../../support/fragments/settings/eholdings';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+
+describe('eHoldings', () => {
+  const testData = {
+    user: {},
+  };
+
+  before('Create test data', () => {
+    cy.createTempUser([Permissions.moduleeHoldingsEnabled.gui]).then((userProperties) => {
+      testData.user = userProperties;
+
+      AssignedUsers.assignUserToDefaultCredentialsViaApi({ userId: testData.user.userId });
```

Comment:
Please remove this line, as assigning user to KB is not necessary when only one KB is configured, and it will revoke access from all unassigned users

Reply:
Fixed